.travis.yml

```diff
@@ -98,3 +136,18 @@ notifications:
     rooms:
       sedfoam:gFgw4hb2TY4uMX10tUA0xb4F#dev
     on_success: always
+© 2020 GitHub, Inc.
+Terms
+Privacy
+
+    Security
+    Status
+    Help
+    Contact GitHub
+    Pricing
+    API
+    Training
+    Blog
+    About
+
+
```

Comment:
i think you can delete the last lines (from  139 to 153)

Reply:
ok! I don't know how I messed it up!